### PR TITLE
tests: benchmarks:  Start testing USBHS with s2ram

### PR DESCRIPTION
**Pull request by nordic-bami**
Verify USB HS driver with s2ram

### CODE REVIEW

File: tests/benchmarks/multicore/idle_usb/Kconfig.sysbuild [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (nordic-segl) [inline tests/benchmarks/multicore/idle_usb/Kconfig.sysbuild:2]: :smile:

File: tests/benchmarks/multicore/idle_usb/testcase.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+common:
+  sysbuild: true
+  depends_on: usbd
+  tags: ci_build ci_tests_benchmarks_multicore usb ppk_power_measure_usb
```
Comment (nordic-piks) [inline tests/benchmarks/multicore/idle_usb/testcase.yaml:4]: Tags needs to stay ppk_power_measure

**Review by nordic-segl — APPROVED** [review]

File: tests/benchmarks/multicore/idle_usb/sysbuild.cmake [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (nordic-segl) [inline tests/benchmarks/multicore/idle_usb/sysbuild.cmake:2]: :smile:

**Review by nordic-pikr — APPROVED** [review]

**Comment by rlubos** [comment]
@nrfconnect/ncs-co-build-system Please review

**Review by nordicjm — APPROVED** [review]

**Pull request merged**